TGC/events/FINflavor.txt

```diff
@@ -25,32 +25,45 @@
 			treasury = -10000
 			random_owned = {
 				limit = {
-					NOT = {
-						province_id = 339
-						province_id = 343
-						province_id = 348
+					OR = {
+						province_id = 346
+						province_id = 344
+						province_id = 341
+						province_id = 342
 					}
 				}
-				add_province_modifier = { name = sixtysix_famine duration = 180 }
+				add_province_modifier = { name = sixtysix_famine_severe duration = 180 }
 			}
 			random_owned = {
 				limit = {
 					NOT = {
+						OR = {
 						province_id = 339
 						province_id = 343
 						province_id = 348
+						province_id = 346
+						province_id = 344
```

Comment:
Somewhat bad indention here, add a extra tab to match the OR scope, this happens in other parts of the code